logging/bin/deploy_fluentbit_azmonitor.sh

```diff
@@ -106,94 +122,96 @@ else
 fi
 
 # Create ConfigMap containing Fluent Bit configuration
-kubectl -n "$LOG_NS" apply -f $FB_CONFIGMAP
+kubectl -n "$LOG_NS" apply -f "$FB_CONFIGMAP"
 
 # Create ConfigMap containing Viya-customized parsers (delete it first)
 kubectl -n "$LOG_NS" delete configmap fbaz-viya-parsers --ignore-not-found
-kubectl -n "$LOG_NS" create configmap fbaz-viya-parsers  --from-file=logging/fb/viya-parsers.conf
+kubectl -n "$LOG_NS" create configmap fbaz-viya-parsers --from-file=logging/fb/viya-parsers.conf
 
 TRACING_ENABLE="${TRACING_ENABLE:-false}"
+tracingValuesFile=""
 if [ "$TRACING_ENABLE" == "true" ]; then
   # Create ConfigMap containing tracing config
   kubectl -n "$LOG_NS" delete configmap fbaz-viya-tracing --ignore-not-found
-  kubectl -n "$LOG_NS" create configmap fbaz-viya-tracing  --from-file=logging/fb/viya-tracing.conf
+  kubectl -n "$LOG_NS" create configmap fbaz-viya-tracing --from-file=logging/fb/viya-tracing.conf
 
   tracingValuesFile="logging/fb/fluent-bit_helm_values_tracing.yaml"
-else 
+else
   # Create empty ConfigMap for tracing since it is expected to exist in main config
   kubectl -n "$LOG_NS" delete configmap fbaz-viya-tracing --ignore-not-found
-  kubectl -n "$LOG_NS" create configmap fbaz-viya-tracing  --from-file="$TMP_DIR"/empty.yaml
+  kubectl -n "$LOG_NS" create configmap fbaz-viya-tracing --from-file="$TMP_DIR"/empty.yaml
 
-  tracingValuesFile=$TMP_DIR/empty.yaml
+  tracingValuesFile="$TMP_DIR/empty.yaml"
 fi
 
 # Check for Kubernetes container runtime log format info
-KUBERNETES_RUNTIME_LOGFMT="${KUBERNETES_RUNTIME_LOGFMT}"
+KUBERNETES_RUNTIME_LOGFMT="${KUBERNETES_RUNTIME_LOGFMT:-}"
```

Comment:
Sets default value to null?

Reply:
it actually sets it to empty string, which I made sure doesn't break the if block on the next line. This is recommended by shellcheck as its considered more robust